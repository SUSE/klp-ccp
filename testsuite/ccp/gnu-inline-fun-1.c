extern inline __attribute__((gnu_inline, always_inline)) void g(void)
{}

extern inline __attribute__((gnu_inline)) void h(void)
{}

void pu_f(void)
{
	g();
	h();
}
