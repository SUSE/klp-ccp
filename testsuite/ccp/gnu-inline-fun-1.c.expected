/* klp-ccp: from testsuite/ccp/gnu-inline-fun-1.c */
extern inline __attribute__((gnu_inline, always_inline)) void g(void)
{}

static inline __attribute__((gnu_inline)) void h(void)
{}

void klpp_pu_f(void)
{
	g();
	h();
}
